manuscript/SAVED.md

```diff
@@ -0,0 +1,40 @@
+Q: How do I get into heaven?
+A: Jesus saves the faithful. So just be faithful to God.
+
+## Commentary ##
+
+We're all going to die one day. Morbid, perhaps, or just real. Death means our physical bodies are going to fail, our hearts will stop, our neurons will stop firing, and life as we know it will come to an end.
+
+But God promises an afterlife. To be clear, God tells is that your soul will live on, heading toward a final judgment at which point you will be either granted eternal life (entry into heaven) or condemned to eternal death (no heaven for you). The only problem is that standard of conduct required for entry is high, unattainable high. So high, in fact, that you and I have already f***ed up.
```

Comment:
Suggestion to not curse or even hint at it. James 3:9, keep this focused on his glory and praise. Suggested replacements: messed up, fowled up, dropped the vall, etc. ;)